docs/release-notes.md

```diff
@@ -12,6 +12,54 @@ Welcome to Semgrep release notes. This document provides an overview of the chan
 
 ## December 2021
 
+### Version 0.77.0
+
+#### Highlights
+
+##### Semgrep CLI and Semgrep CI now ignore the same patterns
+
+With this update, Semgrep CLI now ignores the same patterns as the Semgrep CI by default. Find [the default .semgrepignore on GitHub](https://github.com/returntocorp/semgrep/blob/develop/semgrep/semgrep/templates/.semgrepignore). If you want to return to Semgrep’s previous behavior, create an empty `.semgrepignore` file. However, creating a new `.semgrepignore` overrides the default setup.
+
+##### Autofix improvement
+
+An autofix improvement from [https://github.com/chair6](https://github.com/chair6) from Hashicorp! Big shoutout to them. Fixes several issues (auto fixing multiple things in the same set of lines). This change addresses several issues related to autofix by adding per-file line and column offset tracking, and uses those offsets when making edits to files. The improvement addresses several edge cases in the existing autofix implementation that Semgrep did not handle correctly previously. The addressed issues are the following: [#4428](https://github.com/returntocorp/semgrep/issues/4428), [#3577](https://github.com/returntocorp/semgrep/issues/3577), [#3388](https://github.com/returntocorp/semgrep/issues/3388).
```

Comment:
@pabloest I tried to blow this up a bit. Please check, not sure if this is so important, but I am glad for any additional information.

Reply:
Thanks, @adamkvitek! Looks good to me.

Reply:
Resolving this comment! :-)